spec/app/domain/authentication/authn-oidc/update_input_with_username_from_id_token_spec.rb

```diff
@@ -247,6 +262,108 @@ def request_body(request)
           expect { subject }.to raise_error(::Errors::Authentication::RequestBody::MissingRequestParam)
         end
       end
+
+      context "with valid id token in header" do
```

Comment:
This test also puts the token in the body. The name above implies that it will check the token from the header but that won't be the case. Am I right?

Reply:
We will check token from header. it is missing in bode in this case